cmd/utils/flags.go

```diff
@@ -1803,7 +1813,7 @@ func MakeChainDatabase(ctx *cli.Context, stack *node.Node) ethdb.Database {
 }
 
 func MakeGenesis(ctx *cli.Context) *core.Genesis {
-	var genesis *core.Genesis
+	genesis := core.DefaultGenesisBlock()
```

Comment:
What's the reason for this change? It looks like it previously might have returned `nil` in some cases, and probably error on the caller-side, whereas now it will default to mainnet in those cases. 

Is this a 'safe' change to do?

Reply:
No, that's an artifact of testing

Reply:
Thanks for noticing!